presto-main/src/main/java/com/facebook/presto/dispatcher/LocalDispatchQueryFactory.java

```diff
@@ -67,7 +69,8 @@ public LocalDispatchQueryFactory(
             LocationFactory locationFactory,
             Map<Class<? extends Statement>, QueryExecutionFactory<?>> executionFactories,
             ClusterSizeMonitor clusterSizeMonitor,
-            DispatchExecutor dispatchExecutor)
+            DispatchExecutor dispatchExecutor,
```

Comment:
Nit: can we change the ordering to match the field assignment?  I.e. `clusterSizeMonitor` -> `serverConfig` -> `dispatchExecutor`